Include/Intel/IndustryStandard/ProcessorInfo.h

```diff
@@ -258,7 +258,7 @@ typedef enum {
 #define AMD_CPU_EXT_FAMILY_16H  0x7
 #define AMD_CPU_EXT_FAMILY_17H  0x8
 #define AMD_CPU_EXT_FAMILY_19H  0xA
-#define AMD_CPU_EXT_FAMILY_1AH  0x1A
+#define AMD_CPU_EXT_FAMILY_1AH  0x0B
```

Comment:
Please change to 0xB. How should the changelog be updated?

Reply:
Changed. As for changelog maybe,

Adjusted AMD Family 1Ah CPU support to use `0xB` for `AMD_CPU_EXT_FAMILY_1AH`